pkg/corerp/api/v20231001preview/environment_conversion_test.go

```diff
@@ -269,6 +269,88 @@ func TestConvertVersionedToDataModel(t *testing.T) {
 			},
 			err: nil,
 		},
+		{
+			filename: "environmentresource-terraformrecipe-http.json",
```

Comment:
Isn't this covered by this existing scenario https://github.com/radius-project/radius/blob/9b47d5ea6407e451381fcf8ce971de51f63bae90/pkg/corerp/api/v20231001preview/environment_conversion_test.go#L135

Reply:
Ah yes, removed this and the duplicate tf registry test